src/integrationTest/java/uk/gov/hmcts/darts/utils/MtomLocalClient.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.darts.utils;
+
+import org.junit.jupiter.api.Assertions;
+import uk.gov.hmcts.darts.testutils.IntegrationBase;
+import uk.gov.hmcts.darts.workflow.command.AddAudioMidTierCommand;
+import uk.gov.hmcts.darts.workflow.command.Command;
+import uk.gov.hmcts.darts.workflow.command.CommandFactory;
+
+import java.io.IOException;
+
+public final class MtomLocalClient {
+    private static final int NUMBER_OF_FILE_TRAMSFERS = 1;
+
+    private MtomLocalClient() {
```

Comment:
Do we really need 3 separate applications for doing very similar stuff? Surely this can be combined?

Reply:
I have removed these. This was purely convenience and creates noise